src/telemetry/telemetry.ts

```diff
@@ -19,40 +31,52 @@ class Telemetry implements ITelemetry {
       instrumentationKey: this.getInstrumentationKey(),
       disableExceptionTracking: true,
       disableAjaxTracking: true,
+      disableFetchTracking: false,
```

Comment:
Does this mean we track Fetch information for any call or just calls to Microsoft Graph and the DevX API? If the outer application makes a Fetch call, are we capturing that? What about Fetch to the identity endpoint?

Reply:
Enabling fetch tracking logs all fetch calls. We then drop all but calls to Graph and DevX APIs.

Reply:
It would be helpful to have this commented in the code so that future devs understand that a change to the filtering could result in inadvertent capture of PII.